src/iOS.Essentials/BasicViewExtensions/UIViewExtensions.cs

```diff
@@ -223,4 +223,126 @@ private const int
         view.SetContentHuggingPriority(_lowestPriority, axis);
         return view;
     }
+
+    public static TView ConstraintHeightTo<TView>(this TView view, nfloat constant) where TView : UIView
+    {
+        view.HeightAnchor.ConstraintEqualTo(constant).Active = true;
+        return view;
+    }
+    
+    public static TView ConstraintHeightTo<TView>(this TView view, NSLayoutDimension anchor, nfloat multiplier = default, nfloat constant = default) where TView : UIView
+    {
+        if (multiplier == default)
```

Comment:
What if we set "1" as the default value?

Reply:
I couldn't do it, because appearing error "need using constant value"

Reply:
Hm, let's use float for multiplier then

Reply:
Done!